client_internals_it_test.go

```diff
@@ -47,6 +50,42 @@ func TestListenersAfterClientDisconnected(t *testing.T) {
 	})
 }
 
+func TestNotReceivedInvocation(t *testing.T) {
```

Comment:
If you want to avoid accessing internals and do a test from public API, you can send a long running task to the remote. An EntryProcessor for example which just sleeps. And then close the member.

Reply:
I wonder whether there is an entry processor you mentioned exists in the test jar. I'll try to find that.